src/routers/UpdateSocialMediaRouter.ts

```diff
@@ -0,0 +1,42 @@
+import { Request } from 'express';
+import UserRepo from '../repos/UserRepo';
+import AuthenticatedAppplicationRouter from '../utils/AuthenticatedApplicationRouter';
+import AppDevUtils from '../utils/AppDevUtils';
+
+class UpdateInterestsRouter extends AuthenticatedAppplicationRouter<void> {
+  constructor() {
+    super('POST');
+  }
+
+  getPath(): string {
+    return '/updateSocialMedia/';
+  }
+
+  async content(req: Request): Promise<void> {
+    const { user, body } = req;
+    const { facebook, instagram } = body;
+
+    if (!(facebook || instagram)) {
+      throw Error("At least one of 'facebook' or 'instagram' is required.");
+    }
+
+    let payload = {};
+    if (facebook) {
```

Comment:
Assuming we're going to be introducing other social medias in the future, maybe generalizing this (checking the presence of at least one social media in the body and running a loop where you check if there is a value for a body's key and print out URL provided for <key> is not valid) would prevent us from having to make PRs each time we want to add another social media. Obviously not of essential need atm but those are my 2 cents

Reply:
I didn't think too much about this router tbh just because it will be changing after the MVP (i imagine we should implement FB login, etc instead of having users copy/paste their URLs for the final product), but you're definitely right - gonna eliminate this redundancy rn just because I don't know how long this is gonna stay.